src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -609,7 +612,19 @@ private void OnDiscoveryMessageReceived(ITestDiscoveryEventsHandler2 discoveryEv
             OnDiscoveryAbort(discoveryEventsHandler, ex, false);
         }
     }
-
+    private static bool BlameIsEnabled()
```

Comment:
Personal preference but I like to prefix boolean method with `Is`, `Can` or "action" prefixes. So I would recommend `IsBlameEnabled`.